package.json

```diff
@@ -1,42 +1,51 @@
 {
   "name": "snowplow-tracker",
-  "version": "2.15.0",
+  "version": "2.16.0-M1",
+  "scripts": {
+    "build": "gulp build",
+    "build:all": "(cd core && npm run build) && npm run build",
+    "test:unit": "jest tests/unit/*.spec.js",
+    "test:e2e:sauce": "gulp testEndToEnd && wdio tests/wdio.sauce.conf.js",
+    "test:e2e:local": "gulp testEndToEnd && wdio tests/wdio.local.conf.js"
+  },
   "dependencies": {
     "jstimezonedetect": "1.0.5",
+    "lodash": "^4.17.20",
     "sha1": "git://github.com/pvorb/node-sha1.git#910081c83f3661507d9d89e66e3f38d8b59d5559",
-    "snowplow-tracker-core": "^0.8.0",
+    "snowplow-tracker-core": "file:./core",
```

Comment:
For the prod build & deploy, do we still point to the core directory now rather than the old way of deploying core first then installing it from npm?

Reply:
I've gone back and forth on this a lot. The issue with pointing at a released `core` is that it's really hard to run the e2e tests on saucelabs with any updates to core. It generally makes the whole development process a little fiddly.

I went with this option as ultimately it seems safer too - if someone did make changes to core and didn't push a core release (which would be bad, but possible) then the tracker would still have the latest core code. We should always ensure we do core releases when we update it but this technique means if we ever made a mistake and forgot then we don't bork the JS Tracker.